tests/store/artifact/test_cli.py

```diff
@@ -1,16 +1,27 @@
 import json
```

Comment:
Added some additional test coverage for the existing CLI. Ideally, it would be sufficient to have a single test asserting that this CLI relies on `mlflow.artifacts.download_artifacts()`, but there are some small behavior differences (e.g. for backcompat, the CLI shouldn't throw if run id *and* artifact URI are specified, the CLI also prints the output path). Because these tests are cheap and it's somewhat difficult to test that the CLI calls this method due to the fact that the CLI runs in a subprocess, I've opted to retain existing test coverage and supplement it slightly.

Reply:
Marked for unified logic with 2.0 release? :)

Reply:
Yep! :D